packages/contracts/src/dollar/utils/CollectableDust.sol

```diff
@@ -33,24 +33,25 @@ abstract contract CollectableDust is ICollectableDust {
             _protocolTokens.contains(_token),
             "collectable-dust::token-not-part-of-the-protocol"
         );
-        _protocolTokens.remove(_token);
+        require(_protocolTokens.remove(_token));
```

Comment:
Curious about this added `require`

Reply:
both the EnumerableSet and StructurededLinkList will return a bool for functions that alter the sets. Slither was calling these out as being ignored and a require was the simplest way to ensure a revert on failure.